recipes/approvaltests.cpp/all/conanfile.py

```diff
@@ -45,11 +45,11 @@ def requirements(self):
         if self.options.get_safe("with_boosttest"):
             self.requires("boost/1.72.0")
         if self.options.with_catch2:
-            self.requires("catch2/2.11.0")
+            self.requires("catch2/2.13.7")
         if self.options.with_gtest:
             self.requires("gtest/1.10.0")
         if self.options.with_doctest:
-            self.requires("doctest/2.3.6")
+            self.requires("doctest/2.4.6")
```

Comment:
```suggestion
            self.requires("doctest/2.4.6" if tools.Version(self.version) >= "8.2.0" else "doctest/2.3.6")
```

Reply:
Thank you very much. I thought about a condition like that, but was assuming that the older versions would still fail the CI for armv8 (or whatever the problem platform was).

Do you mean to drop support for older versions of ApprovalTests.cpp @prince-chrismc - I didn't know it was possible to do that...

If so, it solves all these CI problems in stroke, without making the recipe more complex, so is a great idea...